tox.ini

```diff
@@ -73,14 +73,15 @@ commands =
 [testenv:codecov_upload]
 skip_install = true
 skipsdist = true
+passenv =
+    CI
+    CIRCLE*
 deps =
     codecov
 commands =
     codecov \
         --disable search pycov gcov \
         --root girder \
-        --required \
```

Comment:
I think we should keep `--required`, right? Otherwise it fails silently

Reply:
Right.  Added back.